lambda/src/main/scala/pricemigrationengine/handlers/SalesforcePriceRiseCreationHandler.scala

```diff
@@ -0,0 +1,78 @@
+package pricemigrationengine.handlers
+
+import com.amazonaws.services.lambda.runtime.{Context, RequestHandler}
+import pricemigrationengine.model.CohortTableFilter.EstimationComplete
+import pricemigrationengine.model._
+import pricemigrationengine.services._
+import zio.console.Console
+import zio.{App, Runtime, ZEnv, ZIO, ZLayer, console}
+
+case class SalesforcePriceRiseCreationResult()
+
+object SalesforcePriceRiseCreationHandler extends App with RequestHandler[Unit, Unit] {
+
+  val main: ZIO[Logging with Configuration with CohortTable, Failure, Unit] =
+    for {
+      config <- Configuration.config
+      cohortItems <- CohortTable.fetch(EstimationComplete, config.batchSize)
+      _ <- cohortItems.foreach(createSalesforcePriceRise)
+    } yield ()
+
+  private def createSalesforcePriceRise(
+    item: CohortItem
+  ): ZIO[Logging with Configuration with CohortTable, Failure, Unit] =
+    for {
+      config <- Configuration.config
+      result <- updateSalesforce(item)
+        .tapBoth(
+          e => Logging.error(s"Failed to write create Price_Rise in salesforce: $e"),
+          result => Logging.info(s"Estimated result: $result")
+        )
+      _ <- CohortTable
+        .update(result)
+        .tapBoth(
+          e => Logging.error(s"Failed to update Cohort table: $e"),
+          _ => Logging.info(s"Wrote $result to Cohort table")
+        )
```

Comment:
Perhaps update SF and then cohort table for each item

Reply:
Actually, having given it a thumbs up, sorry I dont follow?

Reply:
Sorry, I misread this.  Ignore!